scripts/expert/SimplyJetpacks.zs

```diff
@@ -1,15 +1,64 @@
+import mods.jei.JEI;
+
 #Name: SimplyJetpacks.zs
 #Author: Sander
 #Modpack: Infinity Evolved Reloaded
 #packmode expert
 
 print("Initializing 'SimplyJetpacks.zs'...");
 
-#Replace Truster with Different TE dynamos
-recipes.remove(<simplyjetpacks:metaitemmods:28>);
-recipes.addShaped(<simplyjetpacks:metaitemmods:28>, [[<ore:ingotElectrum>, <thermalfoundation:material:515>, <ore:ingotElectrum>], [<thermaldynamics:duct_0:2>, <thermalexpansion:dynamo:3>, <thermaldynamics:duct_0:2>], [<ore:ingotElectrum>, <forge:bucketfilled>.withTag({FluidName: "redstone", Amount: 1000}), <ore:ingotElectrum>]]);
+// Items
```

Comment:
Remove the vars. They clog up the file.Just use ItemStacks.

Reply:
The problem with itemstacks is that it is really hard to tell what items they are, because mods put a lot of different things under the same item qualifier. It's hard to tell by just reading that `simplyjetpacks:metaitemmods:21` is hardened thruster. The vars uptop is just to give names to them. Like the principle it is better if you can write code that documents itself rather than documenting via the comments. I could use a plain item qualifiers, but it was kinda difficult to figure out which item meant what, when ids changed. For items with descriptive names, sure ItemStacks is the way to go. For items under the same qualifier it's hard.

Just thought having a single goto place for item definitions would be benefitial, rather then going all over the place to change shifted ids.

I'll make one more commit, and if you decide to go with itemstacks only, I'll do so.

Reply:
I understand you concerns, so we can use it. But rearrange your vars, to where you use them and remove those, that you won't use in the same file.

Reply:
Ye, local imports are broken. Even though `/ct syntax` checks out, in runtime `NoClassDefinition` is raised. /shrug